dts/bindings/adc/adi,ad4190-adc.yaml

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2025 Analog Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Bindings for the ADI AD4190 Analog-to-Digital Converter.
+
+compatible: adi,ad4190-adc
+
+include: [adc-controller.yaml, spi-device.yaml]
```

Comment:
Is it correct that ad4170 has device-specific properties but ad4190 and ad4195 don't? It kind of looks like there should be some common ad41xx properties defined and inherited by the more specific part compatibles?

Reply:
The ad4170, ad4190, and ad4195 just share the same device-specific properties. It seems I forgot to add the schema include to inherit properties from common bindings, similar implementation to https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/adc/adi%2Cad4050-adc.yaml#L8 right?